README.md

```diff
@@ -331,33 +285,128 @@ ZkSync Features:
           The factory dependencies in the form `<path>:<contractname>`.
 ```
 
+#### Example
+
+To deploy `src/Greeter.sol` to zkSync testnet:
 
-#### Example Usage
-To Deploy `src/Greeter.sol` to zksync local node:
 ```bash
-../foundry-zksync/target/debug/zkforge zkc src/Greeter.sol:Greeter --constructor-args "ZkSync + Pineapple" --private-key 7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 --rpc-url http://localhost:3050 --chain 270
+../foundry-zksync/target/debug/zkforge zkc src/Greeter.sol:Greeter --constructor-args "ZkSync + Pineapple" --private-key <"PRIVATE_KEY"> --rpc-url https://zksync2-testnet.zksync.dev:443 --chain 280
 ```
 
 #### Output
-```js
+
+```txt
 Deploying contract...
 +-------------------------------------------------+
-Contract successfully deployed to address: 0xa1b809005e589f81de6ef9f48d67e35606c05fc3
-Transaction Hash: 0x34782985ba7c70b6bc4a8eb2b95787baec29356171fdbb18608037a2fcd7eda8
-Gas used: 168141
+Contract successfully deployed to address: 0x07d485ff2df314b240ec392ed86b137a661ddd35
+Transaction Hash: 0xdb6864fe1d19572a3ff509c5c7ed43f033d2dab8261a843808ed46e6e6ee51be
+Gas used: 89879008
 Effective gas price: 250000000
-Block Number: 249
+Block Number: 6651906
 +-------------------------------------------------+
 ```
 
-
 ---
-## Contract Interaction
-***v0.0*** ***Commands***:
-## `zkcast zk-send`
-### aliases: `zkcast zks`, `zkcast zksend`
-Interact with deployed contracts in the native foundry/zkforge fashion using the CLI `zkcast zk-send` command:
+
+## Bridge assets L1 ↔ L2 with `zkcast zk-send` and `zkcast zk-deposit`
+
+### L1 → L2 deposits
+
+```sh
+zkcast zk-deposit <TO> <AMOUNT> <TOKEN> --rpc-url <RPC-URL> --l2-url <L2URL> --chain <CHAIN-ID> --private-key <PRIVATE-KEY>
+```
+NOTE: Leave `<TOKEN>` blank to bridge ETH
+
 ```bash
+Usage: zkcast zk-deposit  <TO> <AMOUNT> --rpc-url <ETH_RPC_URL> --l2-url <L2URL> [OPTIONS] [BRIDGE] [TIP]
+
+Arguments:
+  <TO>
+          The destination of the transaction.
```

Comment:
this is the address on the L2, that will receive the tokens

Reply:
amended